content/en/blog/_posts/2024-05-01-cri-streaming-explained.md

```diff
@@ -114,7 +114,7 @@ sequenceDiagram
     alt Client alternatives
         Note over kubelet,runtime: Container Runtime Interface (CRI)
         kubectl->>API: exec, attach, port-forward
-        API->>kubelet: 
```

Comment:
I assume that the whitespace got trimmed somehow which is causing an syntax error